frigg/builds/models.py

```diff
@@ -310,7 +310,10 @@ def coverage_diff(self):
 
     @property
     def tasks(self):
-        return json.loads(self.result_log)
+        try:
+            return json.loads(self.result_log)
+        except ValueError:
+            return None
```

Comment:
Should we add `logger.exception(error)`? If so I guess it is appropriate to add it to `setup_tasks` as well :)